Gemfile

```diff
@@ -85,7 +85,7 @@ gem "mime-types",                       "~>3.0",             :require => false,
 
 # Modified gems (forked on Github)
 gem "handsoap",                         "=0.2.5.5",          :require => false, :source => "https://rubygems.manageiq.org" # for manageiq-gems-pending only
-gem "ruport",                           "=1.7.0.3",                             :source => "https://rubygems.manageiq.org"
+gem "ruport", :git => "https://github.com/ruport/ruport.git", :branch => "master"
```

Comment:
TODO: Give 🍪 to Keenan to get a new release ☝️

Reply:
We need to release that into rubygems.manageiq.org - I think only me and @bdunne have access to that.

Reply:
Oh wait, does this upgrade to the latest _actual_ ruport release?

Reply:
> Oh wait, does this upgrade to the latest actual ruport release?
Yes, that's the idea.  We started with this:  https://github.com/ruport/ruport/pull/64 and now ride master with it merged.  We'll need to do a release of ruport when @kbrock is ready.

Reply:
`@kbrock merged commit` LOL when did that happen?

Reply:
@kbrock is everywhere you want to be.  #visa

Reply:
@kbrock here is where we need a ruport release